test/hw7/test_task2.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from All_home_works.hw7.task2 import backspace_compare
+
+
+@pytest.mark.parametrize(  # noqa: PT006
+    "first,second,expected",
+    [
+        ("aB#c", "aD#c", True),
+        ("A##c", "#A#c", True),
+        ("#", "", True),
+        ("abcABCD####", "abc", True),
+    ],
+)
+def test__backspace_compare__with_backspace__expected_true(first, second, expected):
```

Comment:
You have `expected_true` in your name. That means that you could simplify this test a bit.

Reply:
I have a question to ask you.
If we have True it's recommended to just assert, but if False or None it's better to use assert False and assert None, isn't it?